1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/features/module/modules/world/Scaffold.java

```diff
@@ -322,10 +322,17 @@ private void place() {
         if (!delayTimer.hasTimePassed(delay) || (sameYValue.get() && launchY - 1 != (int) targetPlace.getVec3().yCoord))
             return;
 
-        int blockSlot = -1;
+        int blockSlot = mc.thePlayer.inventory.currentItem + 36;
```

Comment:
Why? `-1` indicates that there is no block in your hotbar. This is likely to break the code checking if the block was found.